src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,203 @@
+<style scoped>@import url(/app/pages/interest-form/interest-form.css);</style>
+<h1>OpenStax College Interest Form</h1>
+<p>
+    Thank you for your interest in OpenStax College textbooks! Please complete the form below so that we may provide
+    you with the information you need to start saving your students thousands of dollars each semester.
+</p>
+<form accept-charset="UTF-8" action="/adoption_interests" class="new_adoption_interest" method="post">
+    <div style="margin:0;padding:0;display:inline">
+        <input name="utf8" type="hidden" value="✓">
+        <input name="authenticity_token" type="hidden" value="Xi14qRzCbSy2zkso1VEjKU4bEUfFWOrhWbQnh7tlO3c=">
+    </div>
+    <p>Have you already adopted an OpenStax College textbook? <a href="/adoptions">Let us know!</a></p>
+    <h2>Contact Information</h2>
+    <fieldset class="labeled-inputs">
+        <label>
+            <span>First name</span>
+            <input name="adoption_interest[first_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>Last name</span>
+            <input name="adoption_interest[last_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>School name</span>
+            <input name="adoption_interest[school_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>School url</span>
+            <input name="adoption_interest[school_url]" type="text" required="true" />
+        </label>
+        <label>
+            <span>Email address</span>
+            <input name="adoption_interest[email_address]" type="email"
+             pattern="[^@]+@([^@\.]+\.)+[^@\.]+" required="true" />
```

Comment:
Don't use `required="true"`.  `required="false"` also is `true`.  For XHTML you use `required="required"`, although since we're just plan HTML5, you can just truncate it to the property itself: `required`.